Added 'Create New Organization' link to the Settings page

cedricium opened the pull request:
This is the fix for #281, which allows users to click on a link inside the application to create a new organization. Below is the final result based on @rishig's suggestions:

![screen shot 2017-09-14 at 8 57 17 pm](https://user-images.githubusercontent.com/16343560/30465956-5160f8c0-998f-11e7-8a04-6b9779c5bfc2.png)

cedricium committed:
'Create New Organization' added to Settings page

This fixes #281, which will allow users to open an external link in their default
browser to create a new organization on zulipchat.com.

cedricium committed:
Made changes based on @rishig recommendations

Changes include:

- increasing the font size of the 'Create new organization' link
- adding more bottom margin to the link
- aligning the link text with the external navigation icon

akashnimare commented on app/renderer/js/pages/preference/servers-section.js:
```diff
@@ -18,6 +19,8 @@ class ServersSection extends BaseSection {
 				<div id="new-server-container"></div>
 				<div class="title" id="existing-servers"></div>
 				<div id="server-info-container"></div>
+				<!-- <div class="title">Create Organization</div> -->
```
Remove this.

akashnimare commented on app/renderer/js/pages/preference/create-new-org.js:
```diff
@@ -0,0 +1,39 @@
+
+	template() {
+		return `
+			<!-- <div class="settings-card"> -->
```
Either remove the `setting-card` div or uncomment it.

cedricium committed:
Changes made based on review

Removed unnecessary comments and changed the 'Save' button in `new-server-form.js`
to 'Add'.

akashnimare commented:
Merged. Thanks, @cedricium 🚀
